var/spack/repos/builtin/packages/py-agate/package.py

```diff
@@ -16,13 +16,8 @@ class PyAgate(PythonPackage):
 
     license("MIT")
 
-    version("1.6.1", sha256="c93aaa500b439d71e4a5cf088d0006d2ce2c76f1950960c8843114e5f361dfd3")
-
-    depends_on("py-setuptools", type="build")
-    depends_on("py-six@1.9.0:", type=("build", "run"))
-    depends_on("py-pytimeparse@1.1.5:", type=("build", "run"))
-    depends_on("py-parsedatetime@2.1:", type=("build", "run"))
-    depends_on("py-babel@2.0:", type=("build", "run"))
-    depends_on("py-isodate@0.5.4:", type=("build", "run"))
-    depends_on("py-python-slugify@1.2.1:", type=("build", "run"))
-    depends_on("py-leather@0.3.2:", type=("build", "run"))
```

Comment:
No run-time dependencies?

Reply:
Not sure about this one. METADATA file has Requires-dist and has no syntax errors. Just that the info is missing in the PyPI database. 1.6.3 is fine again...